### ISSUE TITLE
A consensus test(TestNodeRunnerPayment) is failed sometimes

### ISSUE THREAD

**Issue by Charleslee522** [bosnet/sebak#147]
```
--- FAIL: TestNodeRunnerPayment (1.78s)
	node_runner_payment_test.go:86: failed to get consensus
```
Failed only in circleci not in local.

**Comment by kfangw** [comment]
There is a timing issue at this part.
https://github.com/bosnet/sebak/blob/master/lib/node_runner_payment_test.go#L48-L65

**Comment by kfangw** [comment]
Same issue might exist in at all of node_runner_*_test
To prevent CI from failing test, this issue should be solved with higher priority.

**Comment by Charleslee522** [comment]
@kfangw What you commented will be solved in #163 :)

**Comment by Charleslee522** [comment]
`TestNodeRunnerPayment` is removed now.
